module-audio/Audio/decoder/decoderFLAC.cpp

```diff
@@ -56,34 +54,37 @@ namespace audio
     void decoderFLAC::setPosition(float pos)
     {
         if (!isInitialized) {
-            LOG_ERROR("Wav decoder not initialized");
+            LOG_ERROR("FLAC decoder not initialized");
             return;
         }
         drflac_seek_to_pcm_frame(flac, flac->totalPCMFrameCount * pos);
 
         // Calculate new position
-        position = float(flac->totalPCMFrameCount) * pos / float(sampleRate);
+        position = static_cast<float>(flac->totalPCMFrameCount) * pos / static_cast<float>(sampleRate);
     }
 
     /* Data encoded in UTF-8 */
-    void decoderFLAC::flac_parse_text(uint8_t *in, uint32_t taglen, uint32_t datalen, uint8_t *out, uint32_t outlen)
+    void decoderFLAC::flac_parse_text(
+        std::uint8_t *in, std::uint32_t taglen, std::uint32_t datalen, std::uint8_t *out, std::uint32_t outlen)
     {
         /* Little Endian here. */
-        uint32_t size = datalen - taglen > outlen - 1 ? outlen - 1 : datalen - taglen;
+        std::uint32_t size = datalen - taglen > outlen - 1 ? outlen - 1 : datalen - taglen;
```

Comment:
In one go, you could also put some parentheses here to make this statement clear.

Reply:
Fixed